test/3x/torch/quantization/test_smooth_quant.py

```diff
@@ -173,6 +180,45 @@ def test_smooth_quant_with_quantize_API(self):
         q_model = quantize(fp32_model, quant_config=quant_config, run_fn=run_fn, example_inputs=example_dict)
         assert q_model is not None, "Quantization failed!"
 
+        quant_config.folding = True
+        fp32_model = copy.deepcopy(model)
+        q_model = quantize(fp32_model, quant_config=quant_config, run_fn=run_fn, example_inputs=example_inputs)
+        assert q_model is not None, "Quantization failed!"
+
+        fp32_model = copy.deepcopy(model)
+        q_model = quantize(fp32_model, quant_config=quant_config, run_fn=run_fn, example_inputs=example_dict)
+        assert q_model is not None, "Quantization failed!"
+
+    @pytest.mark.skipif(not is_ipex_available(), reason="Requires IPEX")
+    def test_sq_save_load_with_quantize_API(self):

```

Comment:
Suggest using prepare/convert to test features

Reply:
SQ's quantize API is kept for ITREX's sq auto alpha, thus need tests to make sure `quantize` can work.